### PR TITLE
chore: bump version

### PR DESCRIPTION
**Pull request by clementsirieix**
## Changes

- Bump the sdk version
- Fix the linter

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ on:
   workflow_dispatch:
   pull_request:
     branches: ["main"]
+  push:
```
Comment (clementsirieix) [inline .github/workflows/CI.yml:7]: I want the linter on the main branch

File: examples/langchain_variable.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 cb = lai.langchain_callback()
 
 # Returns a langchain_openai.ChatOpenAI instance.
-gpt_4o = init_chat_model(
+gpt_4o = init_chat_model(  # type: ignore
```
Comment (clementsirieix) [inline examples/langchain_variable.py:35]: This method is valid as per the doc but langchain has not typed it, ignoring for now.

File: literalai/prompt_engineering/prompt.py [hunk 3]
```diff
@@ -214,9 +214,9 @@ def to_langchain_chat_prompt_template(self, additional_messages=[]):
 
         class CustomChatPromptTemplate(ChatPromptTemplate):
             orig_messages: Optional[List[GenerationMessage]] = Field(
-                default_factory=list
```
Comment (Matthieu-OD) [inline literalai/prompt_engineering/prompt.py:217]: they both do the same thing I would keep list though (it seems easier to read)

**Review by Matthieu-OD — COMMENTED** [review]
Not sure we want to replace `dict` with `lambda: {}` they both do the same but the first option is easier to read

File: literalai/prompt_engineering/prompt.py (hunk 3 above)
Comment (clementsirieix) [reply literalai/prompt_engineering/prompt.py:217]: No they do not as per the linter error: 
```
literalai/observability/generation.py:75: error: Argument "default_factory" to "Field" has incompatible type "type[dict[Any, Any]]"; expected "Union[Callable[[], Never], Callable[[dict[str, Any]], Never]]"  [arg-type]
```

**Review by Matthieu-OD — APPROVED** [review]

**Pull request merged**